README.md

```diff
@@ -64,7 +64,7 @@ For more details on the format and structure of the inputs check out [instructio
 
 Run run.py file from your IDE:
 1. Creating [json file](https://github.com/GeospatialResearch/Digital-Twins/blob/master/src/instructions_linz.json)
-   ```
+   ```python
```

Comment:
``` #!/usr/bin/env python
   ```python
```

Reply:
Similar suggestion - although it doesn't seem to show above I have suggested replacing python with "#!/usr/bin/env python"

Reply:
Hi @Rose, this python here is actually only used to syntax highlight the markdown code. Do you still want to add the shebang? The "```python" will remain and should not change

Reply:
added #!/usr/bin/env python